www.plain.com/docs/_next/static/ddIRkWCFLziDS86DQCfLU/_buildManifest.js

```
@@ -0,0 +1 @@
+self.__BUILD_MANIFEST=function(s,i,a,t,e,n,c){return{__rewrites:{afterFiles:[],beforeFiles:[],fallback:[]},"/404":[s,"static/chunks/pages/404-8dc24a756589c32d.js"],"/500":[s,"static/chunks/pages/500-98b63d51c876a0db.js"],"/_error":["static/chunks/pages/_error-c4bf9dbafd3912bc.js"],"/_sites/[subdomain]/_hidden-login-pages/login":[i,"static/chunks/pages/_sites/[subdomain]/_hidden-login-pages/login-65782e6fd410a448.js"],"/_sites/[subdomain]/_hidden-login-pages/login/jwt-callback":["static/chunks/pages/_sites/[subdomain]/_hidden-login-pages/login/jwt-callback-b7a12bb0457bbd02.js"],"/_sites/[subdomain]/_hidden-login-pages/login/mintlify-callback":["static/chunks/pages/_sites/[subdomain]/_hidden-login-pages/login/mintlify-callback-037ce45206e7cef3.js"],"/_sites/[subdomain]/llms-full.txt":["static/chunks/pages/_sites/[subdomain]/llms-full.txt-5e308440c279171f.js"],"/_sites/[subdomain]/llms.txt":["static/chunks/pages/_sites/[subdomain]/llms.txt-a2b2fec57f6c85ff.js"],"/_sites/[subdomain]/robots.txt":["static/chunks/pages/_sites/[subdomain]/robots.txt-74451a96c200b6d0.js"],"/_sites/[subdomain]/sitemap.xml":["static/chunks/pages/_sites/[subdomain]/sitemap.xml-8296894e4ab6e446.js"],"/_sites/[subdomain]/[[...slug]]":[a,t,e,s,i,n,c,"static/chunks/pages/_sites/[subdomain]/[[...slug]]-79e8424968849281.js"],"/[[...slug]]":[a,t,e,s,i,n,c,"static/chunks/pages/[[...slug]]-19daa314bd4ab490.js"],sortedPages:["/404","/500","/_app","/_error","/_sites/[subdomain]/_hidden-login-pages/login","/_sites/[subdomain]/_hidden-login-pages/login/jwt-callback","/_sites/[subdomain]/_hidden-login-pages/login/mintlify-callback","/_sites/[subdomain]/llms-full.txt","/_sites/[subdomain]/llms.txt","/_sites/[subdomain]/robots.txt","/_sites/[subdomain]/sitemap.xml","/_sites/[subdomain]/[[...slug]]","/[[...slug]]"]}}("static/chunks/9097-597bce25a41f76db.js","static/chunks/7669-b7b6e74eb838f0fc.js","static/chunks/2edb282b-a83f7ffd007bccf0.js","static/chunks/e893f787-f6a1094a35763a0d.js","static/chunks/086d643d-6f7196a364073d16.js","static/chunks/7610-e463a3dc6ba2cbb8.js","static/chunks/40-f2aa17fd5554d7a0.js"),self.__BUILD_MANIFEST_CB&&self.__BUILD_MANIFEST_CB();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)